src/TLuaInterpreter.cpp

```diff
@@ -3099,7 +3099,7 @@ int TLuaInterpreter::setLabelCallback(lua_State* L, const QString& funcName)
     }
 
     QString eventName;
-    if (!lua_isstring(L, 2)) {
+    if (!lua_isstring(L, 1)) {
         lua_pushfstring(L, "%s: bad argument #2 type (event name as string expected, got %s!)", funcName.toUtf8().constData(), luaL_typename(L, 2));
```

Comment:
Should the argument # and index used in `luaL_typename()` error message be changed to 1 as well?

Reply:
It's confusing, but there is a `lua_remove` call above that pops the stack, which is why the 2nd thing becomes 1. The part about `event name` is wrong though...